docs/source/changelog.rst

```diff
@@ -7,7 +7,8 @@ Changelog
 - Drop support for Python 3.4 (:issue:`40`)
 - The dataframe passed to ```.to_gbq(...., if_exists='append')``` needs to contain only a subset of the fields in the BigQuery schema. (:issue:`24`)
 - Use the `google-auth <https://google-auth.readthedocs.io/en/latest/>`__ library for authentication because oauth2client is deprecated. (:issue:`39`)
-- ``read_gbq`` now has a ``auth_local_webserver`` boolean argument for controlling whether to use web server or console flow when getting user credentials. Replaces `--noauth_local_webserver` command line argument (:issue:`35`)
+- ``read_gbq`` now has a ``auth_local_webserver`` boolean argument for controlling whether to use web server or console flow when getting user credentials. Replaces `--noauth_local_webserver` command line argument. (:issue:`35`)
+- read_gbq now displays the BigQuery Job ID in verbose output. (:issue:`70`)
```

Comment:
Please can you add double back ticks around read_gbq, and single back ticks around the issue number? Otherwise, this looks good to me.